docs/src/pages/components/breadcrumbs/RouterBreadcrumbs.tsx

```diff
@@ -104,19 +92,19 @@ export default function RouterBreadcrumbs() {
             );
           }}
         </Route>
-        <nav className={classes.lists} aria-label="mailbox folders">
+        <Navigation aria-label="mailbox folders">
```

Comment:
I think that we should do `<Box component="nav"` here.

As a side note, we have been wondering about changing the API of the Box to `<x.nav sx={{ }} />` .

Reply:
@oliviertassinari  We agree that improving component usage is great. For this case personally from development experience I would not like to have to treat a component as an object. I've never used React components like this except Provider. [Something]. But maybe I'm wrong. Is there a vote or discussion regarding this? to be able to know more about this.

Reply:
> I think that we should do <Box component="nav" here.

Resolved